From 7b07adf752ba8c999cf659f63b458fe98d4a89fb Mon Sep 17 00:00:00 2001
From: valid-ptr <konstantin.matveyev@eligovision.ru>
Date: Thu, 15 Oct 2026 16:56:41 +0300
Subject: [PATCH] Fix C++11 build due to use of std::make_unique

This function is only available in C++14, include our own header
providing its implementation for C++11 in the files using it.

Closes #27229.
---
 src/aui/framemanager.cpp     | 1 +
 src/unix/epolldispatcher.cpp | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/aui/framemanager.cpp b/src/aui/framemanager.cpp
index e8836ef99329..1cefbf6aff29 100644
--- a/src/aui/framemanager.cpp
+++ b/src/aui/framemanager.cpp
@@ -65,6 +65,7 @@ wxDEFINE_EVENT( wxEVT_AUI_FIND_MANAGER, wxAuiManagerEvent );
 #include "wx/dcgraph.h"
 
 #include "wx/generic/private/drawresize.h"
+#include "wx/private/make_unique.h"
 
 #include <map>
 #include <memory>
diff --git a/src/unix/epolldispatcher.cpp b/src/unix/epolldispatcher.cpp
index acf015b06192..c269fb425f23 100644
--- a/src/unix/epolldispatcher.cpp
+++ b/src/unix/epolldispatcher.cpp
@@ -20,6 +20,7 @@
 
 #if wxUSE_EPOLL_DISPATCHER
 
+#include "wx/private/make_unique.h"
 #include "wx/unix/private/epolldispatcher.h"
 #include "wx/unix/private.h"
 #include "wx/stopwatch.h"
